canvas_manage_course/settings/base.py

```diff
@@ -275,6 +264,11 @@
             'level': _DEFAULT_LOG_LEVEL,
             'propagate': False,
         },
+        'django-ssm-parameter-store': {
+            'handlers': ['default', 'console'],
+            'level': _DEFAULT_LOG_LEVEL,
+            'propagate': False,
+        },
```

Comment:
I forget but did we decide on keeping this logging and if so, do we want the console handler?

Reply:
We didn't keep it in caat. removed here as well